main.go

```diff
@@ -71,6 +75,7 @@ type srvInfoResponse struct {
 	PublicKey     string `json:"publicKey"`
 	CurrentEpoch  epoch  `json:"currentEpoch"`
 	NextEpochTime string `json:"nextEpochTime"`
+	MaxPoints     int    `json:"max_points"`
```

Comment:
I prefer snake_case in json, but maybe this should be camelCase to match the other keys?

Reply:
Of course, I wasn't thinking. I just fixed this in a force push. Would you mind stamping the PR again?